src/Rule/Rule.php

```diff
@@ -68,14 +68,16 @@ public function __construct($rule, int $lineNumber = 0, $columnNumber = 0)
     }
 
     /**
+     * @param list<Comment> $commentsBeforeRule
```

Comment:
For this to be consistent on higher PHPStan levels, we also need to change the return type annotation of `ParserState::consumeWhiteSpace()` to be a `list`.

Reply:
Done in #1173.